android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -60,18 +55,23 @@
 import java.util.Locale;
 
 /**
- * This is on boarding activity
- * */
+ * Activity that handles the first run onboarding experience for new Brave browser installations.
+ * Extends FirstRunActivityBase to provide onboarding flows for: - Setting Brave as default browser
+ * - Configuring privacy and analytics preferences (P3A and crash reporting) - Accepting terms of
+ * service The activity guides users through a series of steps using animations and clear UI
+ * elements to explain Brave's key features and privacy-focused approach.
+ */
```

Comment:
```suggestion
 * Activity that handles the first run onboarding experience for new Brave browser installations.
 * Extends FirstRunActivityBase to provide onboarding flows for:
 * - Setting Brave as default browser
 * - Configuring privacy and analytics preferences (P3A and crash reporting)
 * - Accepting terms of service
 * The activity guides users through a series of steps using animations and clear UI
 * elements to explain Brave's key features and privacy-focused approach.
 */
```

Reply:
@mherrmann That's exactly what i had initially but presubmit fails with it so i had to update it. CI was failing with comment's formatting